fedcloud_vm_monitoring/site_monitor.py

```diff
@@ -114,7 +114,7 @@ def get_vm_image_server_show(self, vm_id):
             return "image name not found"
 
     def get_vm_image(self, vm_id, image_name, image_id):
-        if len(image_name) > 0:
+        if (len(image_name) > 0) and ("booted from volume" not in image_name):
```

Comment:
Maybe what you could do is to use `image_name` as last resort option so first the code inside the `else`

Reply:
so first try to get image, then volume and last use image name (even if it's `booted from volume`)

Reply:
Done, please have a look